packages/kbn-test/src/functional_tests/run_tests/flags.ts

```diff
@@ -74,7 +74,7 @@ export function parseFlags(flags: FlagsReader) {
     logsDir: flags.boolean('logToFile')
       ? Path.resolve(REPO_ROOT, 'data/ftr_servers_logs', uuidV4())
       : undefined,
-    esFrom: flags.enum('esFrom', ['snapshot', 'source']) ?? 'snapshot',
+    esFrom: flags.enum('esFrom', ['snapshot', 'source', 'serverless']),
```

Comment:
Doesn't this change remove the defaulting mechanism? Or is it defaulting in all the places where the `esFrom` value is used?

Reply:
There are other places where it will default to `snapshot`. This instance was causing it to always override with `snapshot`, even when later configs had `serverless`.